openshift/kustomize/tekton/base/tasks/deploy-all.yaml

```diff
@@ -117,16 +117,6 @@ spec:
           [env]="dev test prod"
         )
 
-        declare -A COMPONENT5=(
```

Comment:
You will need to resync these values due to the way shell scripts work.

Reply:
I have updated the script. thanks!